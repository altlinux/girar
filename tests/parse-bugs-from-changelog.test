(Closes: 1) (closes 2)
(cLoSeS : #17, #18)
(ALT#1234)
(  cLoSeS : #176  )
( ALT 1)
