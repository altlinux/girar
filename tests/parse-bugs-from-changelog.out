1
2
17
18
19
20
21
22
23
24
25
1234
176
1
0
16
17
13579
